manifests/windows/agent6.pp

```diff
@@ -0,0 +1,64 @@
+# Class: datadog_agent::windows::agent6
+#
+# This class contains the DataDog agent installation mechanism for Windows derivatives
+#
+
+class datadog_agent::windows::agent6(
+  String $service_ensure = 'running',
+  String $baseurl = $datadog_agent::params::agent6_default_repo,
+  String $msi = $datadog_agent::params::datadog_win_msi,
+  String $msi_destination = $datadog_agent::params::win_msi_location,
+  String $api_key = $datadog_agent::api_key,
+  String $hostname = $datadog_agent::host,
+  String $service_name = $datadog_agent::service_name_win,
+  Array  $tags = $datadog_agent::tags,
+  Boolean $service_enable = true,
+  Boolean $should_install = true
+) inherits datadog_agent::params {
+
+	if $should_install {
```

Comment:
Mixed tabs and spaces here and in the matching else below.

Reply:
Ah, gotta love Sublime - ill fix that to all spaces, 2x.

Reply:
Fixed.